src/crc32c/.clang_complete

```diff
@@ -0,0 +1,8 @@
+-Ibuild/include/
+-Ibuild/third_party/glog/
+-Iinclude/
+-Ithird_party/benchmark/include/
+-Ithird_party/googletest/googletest/include/
+-Ithird_party/googletest/googlemock/include/
+-Ithird_party/glog/src/
```

Comment:
`src/Makefile.crc32c.include` doesn't build with glog, so it doesn't need to be included in `contrib/debian/copyright`.